packages/core/tests/helpers.ts

```diff
@@ -25,6 +25,7 @@ import { catchError, filter, map, timeout } from 'rxjs/operators'
 
 import { SubjectInboundTransport } from '../../../tests/transport/SubjectInboundTransport'
 import { SubjectOutboundTransport } from '../../../tests/transport/SubjectOutboundTransport'
+import { BbsModule } from '../../bbs-signatures/src/BbsModule'
```

Comment:
Why do we need the dependency on bbs here? It's fine for now, but ideally we only depend from bbs on core, not vice versa (also in tests)

Reply:
I am using the BbsModule here to set up agent modules in setting up e2e credential tests.

Is there some way to avoid this? Maybe we need to move the agent set up out of helper.ts...??